src/web/RPCServerHandler.h

```diff
@@ -214,28 +214,31 @@ class RPCServerHandler {
                 // if the result is forwarded - just use it as is
                 // if forwarded request has error, for http, error should be in "result"; for ws, error should
                 // be at top
-                if (isForwarded && (json.contains("result") || connection->upgraded)) {
+                if (isForwarded && (json.contains(JS(result)) || connection->upgraded)) {
                     for (auto const& [k, v] : json)
                         response.insert_or_assign(k, v);
                 } else {
-                    response["result"] = json;
+                    response[JS(result)] = json;
                 }
 
                 // for ws there is an additional field "status" in the response,
                 // otherwise the "status" is in the "result" field
                 if (connection->upgraded) {
-                    auto const id = request.contains("id") ? request.at("id") : nullptr;
+                    auto const appendFieldIfExist = [&](auto const& field) {
+                        if (request.contains(field) and not request.at(field).is_null())
```

Comment:
Beautiful. let's apply the same to line 217? will read like english too :)

Reply:
and maybe on 240 too 👍

Reply:
I wish I can replace them , but their logic is slightly different.
For 240: it does not append the same field. 
For 217: appendFieldIfExist will do unnecessary check.

Reply:
I should have explained better that i meant to use `and` instead of `&&` and `or` instead of `||` on those lines too.